main.tf

```diff
@@ -2,32 +2,41 @@ variable "signalfx_key" {
   type = string
 }
 
+terraform {
+  required_providers {
+    signalfx = {
+      source = "splunk-terraform/signalfx"
+      version = "6.1.0"
+    }
+  }
+}
+
 provider "signalfx" {
   auth_token = var.signalfx_key
-  # api_url = "https://api.us0.signalfx.com"
+  # api_url = https://api.us0.signalfx.com"
+}
+
+resource "signalfx_time_chart" "mychart0" {
+    name = "CPU Total Idle"
+    description = "Very cool chart"
+
+    program_text = <<-EOF
+    data("cpu.total.idle").publish()
```

Comment:
hey @daveops-splunk please work with Alastair to make sure this the right SLO metric